flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -296,30 +305,15 @@ def get_output_filename(self, ftype: str, sim_id: int, extension_override: str =
     def get_filename(
         self, ftype: str, sim_id: int, input: bool, extension_override: str = ""
     ):
-        """return a CSP formated filename."""
-
-        if extension_override:  # empty strings are Falsy
-            extension = extension_override
-        else:  # Constructed like this because in some test, extension is not defined
-            extension = self.extension
-
-        if input:
-            run_id = self.in_run_id
-            prefix = self.in_prefix
-        else:
-            run_id = self.out_run_id
-            prefix = self.out_prefix
-
-        fn = self.path_prefix / file_paths.create_file_name(
-            run_id=run_id,
-            prefix=prefix,
-            index=sim_id + self.first_sim_index - 1,
+        return self.path_prefix / file_paths.create_file_name(
```

Comment:
again, let's preserve the argument= style here, to save ourselves future heartache

Reply:
Not sure I see the value since the values match the positional argument name, but done.

Reply:
doesn't look like changes actually stuck here?